WordPressAuthenticator/GoogleSignIn/DataGetting.swift

```diff
@@ -0,0 +1,4 @@
+protocol DataGetting {
+
+    func data(for request: URLRequest) async throws -> Data
+}
```

Comment:
Syntax sugar on top of `URLSession` that is also useful to create a test double for the unit tests.